Add PNP info to PCI attachment of ARCMSR, IICOC, IPMI, ISP, ISCI, IXL, IXLV, IWN, IWI, IDA, IG4, CY drivers

lakhanshiva opened the pull request.

lakhanshiva committed:
Update CY to use PCI_MATCH

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IICOC driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IPMI driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info for PCI attachment of ISP driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info PCI attachments of ISCI driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IXL driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IXLV driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IWN driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IWI driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachment of IDA driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

lakhanshiva committed:
Add PNP info to PCI attachments of IG4 driver

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324

bsdimp commented on sys/dev/ichiic/ig4_pci.c:
```diff
@@ -89,54 +89,98 @@ static int ig4iic_pci_detach(device_t dev);
+
+struct pci_device_table ig4iic_pci_devices[] = {
+	{PCI_DEV(INTEL_VENDOR_ID, PCI_DEVICE_LYNXPT_LP_I2C_1),
+	 PCI_DESCR("Intel Lynx Point-LP I2C Controller-1"), .unused = (1)},
```
All of these should be .driver_data not .unused

bsdimp reviewed (commented):
These look good, minus the 'unused' vs 'driver_data' issue.

lakhanshiva committed:
Update ARCMSR to use PCI_MATCH

Reviewed by: imp, chuck
Submitted by: Lakhan Shiva Kamireddy <lakhanshiva@gmail.com>
Sponsored by: Google, Inc. (GSoC 2018)
Pull Request: https://github.com/bsdimp/freebsd/pull/10
Url: https://reviews.freebsd.org/D16324
